src/pineko/kfactor.py

```diff
@@ -0,0 +1,525 @@
+"""Module to include k_factors in grids."""
+import io
+import pathlib
+
+import lhapdf  # pylint: disable=import-error, import-outside-toplevel
```

Comment:
```suggestion
import lhapdf  # pylint: disable=import-error
```
this is toplevel :upside_down_face: - then why do you need lhapdf?

Reply:
ops just an error :)